script.js

```diff
@@ -1,4 +1,62 @@
+var sciSSorsPaperStone = function () {
+  var randomNumber = Math.floor(Math.random() * 3 + 1);
+  //console.log(sciSSorsPaperStone);
+  if (randomNumber == 1) {
+    return "scissor";
+  }
+  if (randomNumber == 2) {
+    return "paper";
+  }
+  if (randomNumber == 3) {
+    return "stone";
+  }
+};
+
+var sciSSor = "scissor";
+var stoNe = "stone";
+var paPer = "paper";
+var revsciSSor = "reversed scissor";
+var revstoNe = "reversed stone";
+var revpaPer = "reversed paper";
```

Comment:
Same concept of using camelCase applies. It is ok to have 
var scissor = "scissor"
var stone = "stone"
var paper = "paper"
No need to make them different by using different letter casing.